services/notifications.go

```diff
@@ -1675,9 +1675,9 @@ func (n *validatorIsOfflineNotification) GetEpoch() uint64 {
 func (n *validatorIsOfflineNotification) GetInfo(includeUrl bool) string {
 	if n.IsOffline {
 		if includeUrl {
-			return fmt.Sprintf(`Validator <a href="https://%[3]v/validator/%[1]v">%[1]v</a> is offline since epoch <a href="https://%[3]v/epoch/%[2]v">%[2]v</a>).`, n.ValidatorIndex, n.EventEpoch, utils.Config.Frontend.SiteDomain)
+			return fmt.Sprintf(`Validator <a href="https://%[3]v/validator/%[1]v">%[1]v</a> is offline since epoch <a href="https://%[3]v/epoch/%[2]v">%[2]v</a>).`, n.ValidatorIndex, n.EventEpoch-2, utils.Config.Frontend.SiteDomain)
 		} else {
-			return fmt.Sprintf(`Validator %v is offline since epoch %v.`, n.ValidatorIndex, n.EventEpoch)
+			return fmt.Sprintf(`Validator %v is offline since epoch %v.`, n.ValidatorIndex, n.EventEpoch-2)
```

Comment:
**suggestion:** The "epoch - 2" is already saved in `InternalState`, see https://github.com/gobitfly/eth2-beaconchain-explorer/blob/0b59fbe3113e605845949344287bee6aab753216/services/notifications.go#L1559, so we could use that instead of `EventEpoch`.